hub/api/v1/agent_data.py

```diff
@@ -0,0 +1,78 @@
+import sys
+from typing import Any, List, Optional, Union
+
+from fastapi import APIRouter, Depends, HTTPException
+from pydantic import BaseModel
+from sqlmodel import select
+
+from hub.api.v1.auth import AuthToken, get_auth
+from hub.api.v1.models import AgentData, get_session
+
+agent_data_router = APIRouter(
+    tags=["agents, assistants"],
+)
+
+
+class AgentDataRequest(BaseModel):
+    namespace: str
+    name: str
+    key: str
+    value: Union[dict[Any, Any]]
+
+
+def is_hub_account(auth: AuthToken):
+    return True  # todo implement
```

Comment:
Please update this, since we have a functionality to handle Runner Api Key and `is_trusted_runner_api_key()` method